mne_bids/utils.py

```diff
@@ -747,9 +748,9 @@ def _read_events(events_data, event_id, raw, ext):
                              'found %s' % events_data.shape[1])
         events = events_data
     elif 'stim' in raw:
-        events = find_events(raw, min_duration=0.001, initial_event=True)
+        events = find_events(raw, min_duration=0.001, initial_event=True, verbose=verbose)
```

Comment:
check flake8 errors. I suspect this line is too long

Reply:
You are absolutely correct, This and two more lines were too long - fixed.